doc/usage/debugger/debugger.md

```diff
@@ -0,0 +1,9 @@
+ ## Local debugging with VSCode
+ Steps outlining how to run MCAD locally from your IDE / debugger.
+ - Ensure you are authenticated to your Kubernetes/OpenShift Cluster.
+ - Copy the [launch.json.example](https://github.com/project-codeflare/multi-cluster-app-dispatcher/tree/main/doc/usage/debugger/launch.json.example) file and save it as `launch.json` in the `.vscode` folder in the root of your project.
```

Comment:
The file name is `launch.json.template` (not `launch.json.example`)

Reply:
Whoa thanks, excellent catch!